docs/GroupDefs.hpp

```diff
@@ -48,6 +48,48 @@
 /*!
  * \defgroup ControlSystemGroup Control System
  * \brief Contains control system elements
+ *
+ * The control system manages the time-dependent mapping between frames, such as
+ * the fixed computational frame (grid frame) and the inertial frame. The
+ * time-dependent parameters of the mapping are adjusted by a feedback control
+ * system in order to follow the dynamical evolution of the black holes. In BBH
+ * simulations the map is typically a composition of maps that include
+ * translation, rotation, scaling, shape, etc. Each map under the governance of
+ * the control system has an associated time-dependent map parameter
+ * \f$\lambda(t)\f$ that is regularly updated by a control signal \f$U(t)\f$,
+ * which is determined by an error function \f$Q(t)\f$ that is uniquely
+ * specified for each map. The control system is designed to minimize the error,
+ * \f$Q(t) \rightarrow 0\f$. This is accomplished by updating the Nth derivative
+ * of the piecewise polynomial FunctionOfTime associated with the parameter
+ * \f$\lambda(t)\f$. Note that \f$\lambda(t)\f$, \f$U(t)\f$, and \f$Q(t)\f$ can
+ * be vectors.
+ *
+ *
+ * The key components of the control system are:
+ * - FunctionsOfTime: each map has an associated FunctionOfTime which holds the
+ *   map parameter \f$\lambda(t)\f$ and relevant time derivatives.
+ * - ControlError: each map has an associated ControlError which computes
+ *   the error, \f$Q(t)\f$, in the mapping.
+ * - Averager: an averager can be used to average out the noise in the 'raw'
+ *   \f$Q(t)\f$ returned by the ControlError.
+ * - Controller: the map controller computes the control signal \f$U(t)\f$ from
+ *   the computed \f$Q(t)\f$ and, depending on the controller type, some
+ *   combination of the integral of \f$Q(t)\f$ and/or its derivatives. Two
+ *   common controllers are PID (proportional/integral/derivative)
```

Comment:
How about:

Controller: the map controller computes the control signal U(t) from Q(t) and its time integral and time derivatives. The control is accomplished by setting the Nth derivative of lambda(t) to U(t). Two common controllers are...